server/questions/views.py

```diff
@@ -62,7 +62,7 @@ def get_queryset(self):
 
 
 class QuestionListView(generics.ListAPIView):
-    permission_classes = [permissions.IsAuthenticated]
+    permission_classes = [IsAdmin]
```

Comment:
@ShawnCollinge is this okay?

Reply:
or do we want users to be able to view through questions api

Reply:
I think they need to have access to see a single question don't they? I'll take a look at the code later.

Reply:
ah that used to be the case but we are hydrating the interview record with the question server side now iirc

Reply:
this is something we can just as easily see as admins by querying the DB. unless someone get's admin access then it shouldn't be shown though